src/createAnimatedComponent/createAnimatedComponent.tsx

```diff
@@ -244,9 +244,7 @@ export function createAnimatedComponent(
     _detachStyles() {
       if (IS_WEB && this._styles !== null) {
         for (const style of this._styles) {
-          if (style?.viewsRef) {
-            style.viewsRef.remove(this);
-          }
+          style.viewsRef.delete(this);
```

Comment:
Shouldn't it be `style.viewsRef?.delete(this);` here instead of in line 40?

Reply:
No need to use `?` here since we have `IS_WEB` check above.